lib/task.js

```diff
@@ -77,17 +78,8 @@ Create a list of cached volumes that will be mounted within the docker container
 @param {object} volumes to mount in the container
  */
 async function buildVolumeBindings(taskVolumeBindings, volumeCache, taskScopes) {
-  let neededScopes = [];
-
-  for (let volumeName in taskVolumeBindings) {
-    neededScopes.push('docker-worker:cache:' + volumeName);
-
-    if (!scopeMatch(taskScopes, neededScopes)) {
-      throw new Error(
-        'Insufficient scopes to attach "' + volumeName + '" as a cached ' +
-        'volume.  Try adding ' + neededScopes + ' to the .scopes array.'
-      );
-    }
+  if (!hasPrefixedScopes('docker-worker:cache:', taskVolumeBindings, taskScopes) {
```

Comment:
should be if (!hasPrefixedScopes('docker-worker:cache:', taskVolumeBindings, taskScopes)) {